iceberg-catalog-migrator/cli/src/test/java/org/apache/polaris/iceberg/catalog/migrator/cli/AbstractCLIMigrationTest.java

```diff
@@ -116,12 +114,6 @@ protected void beforeEach() {
 
   @AfterEach
   protected void afterEach() {
-    // manually refreshing catalog due to missing refresh in Nessie catalog
-    // https://github.com/apache/iceberg/pull/6789
-    // create table will call refresh internally.
-    sourceCatalog.createTable(TableIdentifier.of(BAR, "tblx"), schema).refresh();
```

Comment:
Since for polaris, source and target catalog uses same warehouse (Needed for migration). This was failing in CI when target catalog was trying to create table as the same path table exists. 

Now. I removed this as the above mentioned Nessie issue (in the comment) is fixed and we don't need this workaround.